internal/bminventory/inventory_test.go

```diff
@@ -10018,6 +10018,110 @@ var _ = Describe("TestRegisterCluster", func() {
+				DesiredNetworkType:   models.ClusterCreateParamsNetworkTypeOpenShiftSDN,
+				Platform:             models.Platform{Type: models.NewPlatformType(models.PlatformTypeBaremetal)},
+			},
+			// OpenShift version < 4.12.0-0.0, SNO => OVNKubernetes
+			{
+				OpenShiftVersion:     swag.String("4.11.17"),
+				HighAvailabilityMode: swag.String(models.ClusterCreateParamsHighAvailabilityModeNone),
+				DesiredNetworkType:   models.ClusterCreateParamsNetworkTypeOVNKubernetes,
+				Platform:             models.Platform{Type: models.NewPlatformType(models.PlatformTypeNone)},
+			},
+			// OpenShift version >= 4.12.0-0.0, HighAvailability = nil => OVNKubernetes
+			{
+				OpenShiftVersion:     swag.String("4.13.2"),
+				HighAvailabilityMode: nil,
+				DesiredNetworkType:   models.ClusterCreateParamsNetworkTypeOVNKubernetes,
+				Platform:             models.Platform{Type: models.NewPlatformType(models.PlatformTypeBaremetal)},
+			},
+			// OpenShift version < 4.12.0-0.0, HighAvailability = nil => OpenShiftSDN
+			{
+				OpenShiftVersion:     swag.String("4.8.5"),
+				HighAvailabilityMode: nil,
+				DesiredNetworkType:   models.ClusterCreateParamsNetworkTypeOpenShiftSDN,
+				Platform:             models.Platform{Type: models.NewPlatformType(models.PlatformTypeBaremetal)},
+			},
+			// OpenShift version >= 4.12.0-0.0, SNO, IPv4 => OVNKubernetes
+			{
+				OpenShiftVersion:     swag.String("4.12.0-ec.3"),
+				HighAvailabilityMode: swag.String(models.ClusterCreateParamsHighAvailabilityModeNone),
+				DesiredNetworkType:   models.ClusterCreateParamsNetworkTypeOVNKubernetes,
+				Platform:             models.Platform{Type: models.NewPlatformType(models.PlatformTypeNone)},
+			},
+			// OpenShift version < 4.12.0-0.0, SNO => OVNKubernetes
+			{
+				OpenShiftVersion:     swag.String("4.11.0-rc.3"),
+				HighAvailabilityMode: swag.String(models.ClusterCreateParamsHighAvailabilityModeNone),
+				DesiredNetworkType:   models.ClusterCreateParamsNetworkTypeOVNKubernetes,
+				Platform:             models.Platform{Type: models.NewPlatformType(models.PlatformTypeNone)},
+			},
+			// OpenShift version >= 4.12.0-0.0, SNO => OVNKubernetes
+			{
+				OpenShiftVersion:     swag.String("4.12-fc.2"),
+				HighAvailabilityMode: swag.String(models.ClusterCreateParamsHighAvailabilityModeNone),
+				DesiredNetworkType:   models.ClusterCreateParamsNetworkTypeOVNKubernetes,
+				Platform:             models.Platform{Type: models.NewPlatformType(models.PlatformTypeNone)},
+			},
+			// OpenShift version >= 4.12.0-0.0, SNO => OVNKubernetes
+			{
+				OpenShiftVersion:     swag.String("4.12-rc.1"),
+				HighAvailabilityMode: swag.String(models.ClusterCreateParamsHighAvailabilityModeNone),
+				DesiredNetworkType:   models.ClusterCreateParamsNetworkTypeOVNKubernetes,
+				Platform:             models.Platform{Type: models.NewPlatformType(models.PlatformTypeNone)},
+			},
+		}
+
+		for _, t := range tests {
```

Comment:
If you add a `Name` parameter to `clusterParamsForTest` (initialized with what is currently simple `//` comments) and put the `It(...)` call inside this loop body, you can pass `t.Name` to `It` and so it would show up as nicer separate tests in the unit test reports